custom_components/nobo_hub/climate.py

```diff
@@ -45,20 +59,12 @@
     HVAC_MODE_HEAT, HVAC_MODE_AUTO
 ]
 
-MIN_TEMPERATURE = 7
-MAX_TEMPERATURE = 40
-
 _LOGGER = logging.getLogger(__name__)
 
-_ZONE_NORMAL_WEEK_LIST_SCHEMA = vol.Schema({cv.string: cv.string})
 
-# Validation of the user's configuration
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_HOST): cv.string,
-    vol.Optional(CONF_IP_ADDRESS, default='discover'): cv.string,
-    vol.Optional(CONF_COMMAND_OFF, default=''): cv.string,
-    vol.Optional(CONF_COMMAND_ON, default={}): _ZONE_NORMAL_WEEK_LIST_SCHEMA,
-})
+def clean_up(name):
```

Comment:
clean_up is a bit too generic name
```suggestion
def replace_nbsp(name):
```